OxfordBodleian/Juel-Jensen/BDLaethg36.xml

```diff
@@ -102,90 +95,304 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                        </supportDesc>
+                    
+                        <layoutDesc>
+        <layout columns="1" writtenLines="12">
+            <locus from="1" to="52"></locus>
+            <locus target="#54"></locus>
+            <note corresp="#textarea #margin">Data on text area and margin dimensions taken from <locus target="#33r"/>.</note>
+            <dimensions unit="mm" xml:id="textarea">
+                <height>73</height>
+                <width>55</width>
+            </dimensions>
+            <dimensions type="margin" unit="mm" xml:id="margin">
+                <dim type="top">5</dim>
+                <dim type="bottom">15</dim>
+                <dim type="right">5</dim>
+                <dim type="left">10</dim>
+            </dimensions>
+            <note>There are 8–12 characters per line.</note>
+            <ab type="pricking">Ruling and pricking are visible. Primary pricks are visible and ruling pricks are mostly visible.</ab>
+            <ab type="ruling" subtype="pattern">
+                Ruling pattern: 1E!-1E!/0/0/B.</ab>
+            <ab type="ruling">The upper line is mostly written above the
+                ruling, but occasionally under it (e.g. <locus from="41" to="44"/>). The bottom line is written above the
+                ruling.</ab>
+        </layout>
+    </layoutDesc>
+                    </objectDesc>
+                    
+                    <handDesc>
+                        <handNote xml:id="h1" script="Ethiopic">
+                            <locus from="1r" to="52v"/>
+                            <locus from="54r" to="54v"/>
+                            <seg type="ink">Black and red.</seg>
+                            <date notBefore="1850" notAfter="1950"></date>
+                            <desc>
+                                Irregular hand, but characters are regularly spaced.
+                            </desc>
+                            <seg type="rubrication">
+                                Initial introductory formula; two lines on the incipit page <locus target="#40r"/>; 
+                                the initial word(s) of sections of prayers 
+                                (e.g. <foreign xml:lang="gez">አምሕለከ፡</foreign> or an abbreviation thereof 
+                                on <locus from="3v" to="12r"/>; <foreign xml:lang="gez">መሐልኩ፡ ለኪ፡</foreign> or an abbreviation thereof 
+                                on <locus from="33r" to="38r"/>; different words on <locus from="40v" to="54r"/>); holy names; 
+                                names of owners; elements of numerals and punctuation signs.
+                                </seg>
+                        </handNote>
+                    </handDesc>
+                    
+                    <decoDesc>
+                        <decoNote type="ornamentation" xml:id="d1">
+                            <locus target="#40r"></locus>
+<!--                            <desc> Jacopo
```

Comment:
"Single band filled with a pattern of interlocked black and red triangles"?

Reply:
Thanks! I add it